report_py3o/models/ir_actions_report.py

```diff
@@ -194,3 +194,28 @@ def _get_attachments(self, res_ids):
                     if attachment_id:
                         save_in_attachment[record_id.id] = attachment_id
         return save_in_attachment
+
+    @api.model
+    def _get_rendering_context_model(self):
+        report_model_name = 'report.%s' % self.report_name
+        return self.env.get(report_model_name)
+
+    @api.model
+    def _get_rendering_context(self, docids, data):
```

Comment:
@tbaden Why do we need these methods? These are never called by the current addon. (and seems not to be part of the odoo api since they don't call `super`)

Reply:
this is called here:

https://github.com/OCA/reporting-engine/blob/4ce795a6d0ef9ea40c0fd00a6aa83d592e362e7b/report_py3o/models/py3o_report.py#L197-L199

in v12 these are functions in odoo core. v11 don't have these. in v10 this module hab a custom function using rml_parse.